pyvo/registry/regtap.py

```diff
@@ -873,10 +873,10 @@ def _build_vosi_column(self, column_row):
 
     def _build_vosi_table(self, table_row, columns):
         """
-        return a io.vosi.vodataservice.Table element for a
+        return a io.vosi.vodataservice.VODataServiceTable element for a
         query result from get_tables.
         """
-        res = vodataservice.Table()
+        res = vodataservice.VODataServiceTable()
```

Comment:
Hm.  I wonder why the coverage machinery considers this uncovered.  I've just checked: all of test_regtap.TestGetTables fail if you get this wrong.  I won't debug codecov, but I consider this warning a false positive.

Reply:
codecov is sometimes behaves weird. My usual approach for it is not to take it too seriously, but if it indicates something substantial, look into the details manually.
Thank you for doing it for this case.